examples/pytorch/text-classification/run_glue_no_trainer.py

```diff
@@ -188,6 +196,18 @@ def main():
     if args.seed is not None:
         set_seed(args.seed)
 
+    # Handle the repository creation
+    if accelerator.is_main_process:
+        if args.push_to_hub:
+            if args.hub_model_id is None:
+                repo_name = get_full_repo_name(Path(args.output_dir).name, token=args.hub_token)
```

Comment:
Does the `args.hub_token` always have to be passed or is the locally saved fetched token taken in case `args.hub_token` is None? 

Also is a nice error message thrown when the token is incorrect? Since `push_to_hub` now defaults to True - I think it'd be important to provide a nice error message and maybe also how to disable saving if wished

Reply:
> Does the args.hub_token always have to be passed or is the locally saved fetched token taken in case args.hub_token is None?

Yes, it uses the local token if none is passed.

Note that the default to `push_to_hub` is not True, a user has to pass it. I'll add in the README that they should execute `huggingface-cli login` before running the script. Will also look at the error message and open an issue on `huggingface_hub` if it's not clear enough.